tools/single_use/Normalise_Consent_Data.php

```diff
@@ -239,7 +239,25 @@
 
 // Populate candidate_consent_rel
 foreach ($dataArray as $consentValues) {
-    $db->insert('candidate_consent_rel', $consentValues);
+    $candidateID    = $consentValues['CandidateID'];
```

Comment:
why not do this in the validation code above ??

Reply:
😑

Reply:
because you require the `consent` table to be populated before doing this validation

Reply:
why not check in the validation if ANY data is there
?